package.json

```diff
@@ -7,19 +7,17 @@
   "repository": "adobe/aio-cli-plugin-asset-compute",
   "dependencies": {
     "@adobe/aio-cli-plugin-runtime": "^1.6.0",
-    "@adobe/asset-compute-devtool": "^1.0.1",
+    "@adobe/asset-compute-devtool": "^2.0.0",
     "@oclif/command": "^1.5.19",
     "@oclif/config": "^1.14.0",
     "@oclif/plugin-help": "^2.2.3",
     "amazon-s3-uri": "0.0.3",
     "cachedir": "^2.3.0",
+    "dotenv": "^8.2.0",
```

Comment:
this is actually not needed since it is loaded in the index.js in the devtool: https://github.com/adobe/asset-compute-devtool/blob/master/server/index.js#L17